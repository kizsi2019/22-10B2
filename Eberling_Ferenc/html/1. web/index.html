<!DOCTYPE html>
<html>
<head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="bootstrap.min.css">
    <script src="jquery.min.js"></script>
</head>

<body>
    <nav class="sidebar navbar navbar-expand-lg navbar-dark bg-dark d-lg-block">
        <h1 class="navbar-brand">
            FORMULA ONE
        </h1>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false"
            aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="navbarNavDropdown">
            <ul class="navbar-nav flex-column">
                <li class="nav-item active">
                    <a class="nav-link text-white" href="#home">Kezdőlap</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link text-white" href="#records">"Leg"-ek</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link text-white" href="#avgspeed">Átlagsebességek</a>
                </li>
            </ul>
        </div>
    </nav>

    <main class="main-content">
        <section id="home" class="section-100 bg-image">
            <h1 class="text-center">Érdekességek a Forma-1 világából</h1>
        </section>

        <section id="history">
        </section>

        <section id="records">
            <div class="row mx-0">
                <div class="bg-black col-sm-12 col-md-6 p-3">
                    <h3 class="bg-dark p-2 col-sm-12 text-center">
                        Legtöbb nagydíj
                    </h3>
                    <div class="row">
                        <img class="col-sm-4" src="img/rubens_barrichello.jpg" alt="Rubens Barrichello">
                        <div class="col-sm-8">
                            <ol>
                                <li>Rubens Barrichello (326)</li>
                                <li>Kimi Räikkönen (315)</li>
                                <li>Fernando Alonso (314)</li>
                                <li>Jenson Button (309)</li>
                                <li>Michael Schumacher (308)</li>
                                <li>Felipe Massa (272)</li>
                            </ol>
                        </div>
                    </div>
                </div>

                <div class="bg-black col-sm-12 col-md-6 p-3">
                    <h3 class="bg-dark p-2 col-sm-12 text-center">
                        Legtöbb megnyert futam
                    </h3>
                    <div class="row">
                        <img class="col-sm-4" src="img/michael_schumacher.jpg" alt="Michael Schumacher">
                        <div class="col-sm-8">
                            <ol>
                                <li>Michael Schumacher (91)</li>
                                <li>Lewis Hamilton (84)</li>
                                <li>Sebastian Vettel (53)</li>
                                <li>Alain Prost (51)</li>
                                <li>Ayrton Senna (41)</li>
                                <li>Fernando Alonso (32)</li>
                            </ol>
                        </div>
                    </div>
                </div>

                <div class="bg-black col-sm-12 col-md-6 p-3">
                    <h3 class="bg-dark p-2 col-sm-12 text-center">
                        Bajnoki címek
                    </h3>
                    <div class="row">
                        <img class="col-sm-4" src="img/juan_manuel-fangio.jpg" alt="Juan Manuel Fangio">
                        <div class="col-sm-8">
                            <ol>
                                <li>Michael Schumacher (7)</li>
                                <li>Lewis Hamilton (6)</li>
                                <li>Juan Manuel Fangio (5)</li>
                                <li>
                                    Alain Prost (4)
                                    <br>
                                    Sebastian Vettel (4)
                                </li>
                            </ol>
                        </div>
                    </div>
                </div>

                <div class="bg-black col-sm-12 col-md-6 p-3">
                    <h3 class="bg-dark p-2 col-sm-12 text-center">
                        Csapat bajnoki címek
                    </h3>
                    <div class="row">
                        <img class="col-sm-4" src="img/ferrari.jpg" alt="Ferrari">
                        <div class="col-sm-8">
                            <ol>
                                <li>Ferrari (16)</li>
                                <li>Williams (9)</li>
                                <li>McLaren (8)</li>
                                <li>Lotus (7)</li>
                                <li>Mercedes (6)</li>
                                <li>Red Bull (4)</li>
                            </ol>
                        </div>
                    </div>
                </div>

                <!-- Legfiatalabb versenyző -->

                <!-- Legidősebb versenyző -->

                <!-- Legtöbb Grand Prix -->


            </div>
        </section>

        <section id="avgspeed" class="section-100 bg-image p-2">
            <h2 class="text-center py-3">Átlagsebesség számítás</h2>
            <div class="row mx-0">
                <div class=".d-none .d-md-block col-md-5">
                    <img id="circuitimage" class="w-100">
                </div>

                <form class="col-md-5 offset-md-1">
                    <div class="form-group">
                        <label for="circuit">Pálya</label>
                        <select class="form-control" id="circuit" oninput="calculate()">
                            <option value=""></option>
                            <option value="HUN">Hungaroring</option>
                            <option value="MON">Monaco</option>
                            <option value="BEL">Spa-Francorchamps</option>
                        </select>
                    </div>
                    <div class="form-group">
                        <label for="laptime">1 kör ideje (másodpercben)</label>
                        <input type="text" class="form-control" id="laptime" placeholder="pl.: 105.954" oninput="calculate()">
                    </div>

                    <div class="form-group">
                        <label for="averagespeed">Átlagsebesség</label>
                        <input type="text" class="form-control" id="averagespeed" readonly>
                    </div>
                </form>

                <script>
                </script>

            </div>

        </section>
    </main>

    <audio id="f1sound">
        <source src="fromula-one-car-sound.mp3" type="audio/mpeg">
    </audio>

    <footer class="text-right p-2">
        Források: http://formula.uw.hu/f1tortenet.html, https://en.wikipedia.org/wiki/List_of_Formula_One_driver_records
    </footer>


    <script src="popper.min.js"></script>
    <script src="bootstrap.min.js"></script>
</body>

</html>