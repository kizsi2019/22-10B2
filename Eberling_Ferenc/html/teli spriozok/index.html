<!DOCTYPE html>
<html lang="hu">

<head>
    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
    <title>Téli sportok</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- Bootstrap stíluslap hivatkozás -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
    <!-- Saját stíluslap hivatkozás -->

</head>

<body>
    <!-- Fejléc -->
    <header class="container text-center text-white p-5">
        <h1>Téli sportok</h1>
        <p>A téli sportok olyan sportágak és sporttevékenységek összefoglaló nevei, amelyeket télen vagy mesterségesen létrehozott téli körülmények között lehet űzni.</p>
    </header>

    <!-- Menü -->
    <nav class="navbar navbar-expand-lg navbar-light bg-light sticky-top">
        <div class="container">
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>

            <div class="collapse navbar-collapse justify-content-center" id="navbarSupportedContent">
                <ul class="navbar-nav mr-auto mb-2 mb-lg-0 ">
                    <!-- Aktív menüpont -->
                    <li class="nav-item active"><a class="nav-link" href="#">Kezdőlap</a></li>
                    <!-- További menüpontok -->
                    <li class="nav-item"><a class="nav-link" href="sieles.html">Síelés</a></li>
                    <!-- Új menüpont -->

                </ul>
            </div>
        </div>
    </nav>

    <!-- Tartalom -->
    <main class="container p-3">

        <!-- Bevezető szöveg -->
        <section id="bevezeto">

            <!-- Cím -->
            <h2>A téli sportok története</h2>

            <!-- Szöveg -->
            A téli sportok története nagyon régre nyúlik vissza. A legrégebbi bizonyíték a síelésre egy 8000 éves síléc, amelyet Svédországban találtak. A korcsolyázásra utaló első írásos emlék az ókori görögök idejéből származik, akik csontdarabokat kötöttek a lábukra
            a jégen való csúszkáláshoz. A szánkózás és a szánhúzás már az ókorban is elterjedt volt Európa és Ázsia egyes területein. A jégkorongozást először az 1700-as években említették Kanadában, mint egy angol katonák által játszott játékot. A modern
            téli sportok kialakulása a 19. században kezdődött meg, amikor több európai országban megalakultak az első síklubok és síversenyeket rendeztek. Az első téli olimpiát 1924-ben tartották Franciaországban, ahol 16 ország vett részt 6 sportágban:
            Alpesi sí Műkorcsolya Gyorskorcsolya Jégkorong Síelés Műugrás

        </section>

        <!-- Képgaléria -->
        <section id="kepgaleria">

            <!-- Cím -->
            <h2>Képgaléria</h2>

            <!-- Képek -->
            <div class="row">
                <div class="col-sm-6 col-md-4 col-lg-3 p-2">
                    <img src="images/sielo.jpg" alt="Síelő">
                </div>
                <div class="col-sm-6 col-md-4 col-lg-3 p-2">
                    <img src="images/korcsolyazo.jpg" alt="Korcsolyázó">
                </div>
                <div class="col-sm-6 col-md-4 col-lg-3 p-2">
                    <img src="images/jegkorongozo.jpg" alt="Jégkorongozó">
                </div>
                <div class="col-sm-6 col-md-4 col-lg-3 p-2">
                    <img src="images/snowboardozo.jpg" alt="Snowboardozó">
                </div>
            </div>

        </section>

    </main>

    <!-- Lábléc -->
    <footer class="container text-center bg-dark text-white p-3">

        <!-- Szöveg -->
        &copy; 2022 Téli sportok

    </footer>

    <!-- Bootstrap szkript hivatkozás -->
    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>

</body>

</html>