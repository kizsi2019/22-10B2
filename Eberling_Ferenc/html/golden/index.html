<!DOCTYPE html>
<html lang="hu">
<head>
    <meta charset="UTF-8">
    <link rel="stylesheet" type="text/css" href="css/goldi.css">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Golden Retrieverek - Kezdőlap</title>
</head>
<body>
<p class="menu"><a href="index.html">Kezdőlap</a> 
<a href="tipusok.html"> Típusok</a></p>
<h1 id="eleje">A golden retriever bemutatása</h1>
<h2>Előnyös tulajdonságai</h2>
<h2>Goldi, a kutyám</h2>
<p>A <b>golden retriever</b> egy kedves, barátságos kutyafajta,
mely Skóciából származik. <br>A fajta tudományos megnevezése:
golden retriever - <i>Canis lupus familiaris.</i></p>

<p><strong>Fontos!</strong> Ha gazdi szeretnél lenni, nézz utána,
hogy a <em>felelős</em> kutyatartással kapcsolatban milyen ajánlások
és kötelezettségek vannak!</p>
<ul>
    <li>Okos, könnyen képezhető</li>
    <li>Szelíd, családbarát</li>
    <li>Nagyon jó vadászkutya</li>
</ul>
<img src="img/goldi.jpg" height="200" alt="A kutyám, aki egy golden retriever." title="Goldi, a kutyám (felnőttkorában)">
<img src="img/kolyok.jpg" height="200" alt="A kutyám kölyökkorában." title="Goldi, a kutyám (kölyökkorában)">
<p>Ha szeretnél még több kutyás fotót nézegetni, látogasd meg a
<a href="http://tiny.cc/goldipix">Pixabay portál - Goldenek</a>
fotó oldalát!</p>   
<video controls width="400">
    <source src="./video/goldiseta.mp4" type="video/mp4">
</video> 
<p>A videón is láthatod, hogy Goldi nagyon szeret a földön hemperegni, amely miatt általában koszosan jövünk haza a sétából.</p>
   
<p><a href="#eleje">Vissza az oldal tetejére</a></p>

</body>
</html>